src/client/arg_parse.h

```diff
@@ -0,0 +1,13 @@
+#ifndef CLIENT_HEADER
+#define CLIENT_HEADER
+
+    struct return_struct
+	{
+		int8_t cmd;
+		int64_t payload_len;
+		char *payload;
```

Comment:
Arg parsing shouldn't be aware of IPC stuff like payload.

The arg values are either a path for `CMD_ADD` or a job ID for the other commands. I propose we use a union for the possible values, and have `cmd` as the discriminator.
```c
union {
    int64_t job_id;
    struct { int64_t path_len; char *path; };
};
```